src/main/java/com/gachigang/ontherun/controller/ProductController.java

```diff
@@ -0,0 +1,65 @@
+package com.gachigang.ontherun.controller;
+
+import com.gachigang.ontherun.common.mapper.ProductMapper;
+import com.gachigang.ontherun.payload.product.request.CreateProductRequest;
+import com.gachigang.ontherun.payload.product.response.ProductResponse;
+import com.gachigang.ontherun.service.ProductService;
+import jakarta.validation.Valid;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.data.domain.Pageable;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequiredArgsConstructor
+@Slf4j
```

Comment:
I dont thing that we need it here, since we dont log anything

Reply:
Already fixed it